### DISCUSSION TITLE
New Akamai Challange on login

### DISCUSSION THREAD

**Discussion by ghost** [timdorr/tesla-api#540] [Q&A]
I'm trying to use `puppeteer` to automate the sign-in process for Tesla. 
Following method shown by @SystemDisc in #501 , I tried setting up a `puppeteer` environment using the same code. 

I keep hitting a 30 second challenge, and then it takes me back to the email input screen. 
![image](https://user-images.githubusercontent.com/101129855/157078591-cfff2a42-347b-4a24-8c1a-5d56355d625a.png)

Any way to skip this screen?

I can use tool like 2Captcha to bypass the reCaptcha, but this 30 sec wait is annoying :(

---

Here is the whole flow : 
![tesla_login_fast](https://user-images.githubusercontent.com/101129855/157081156-77055bb5-5ef1-432e-bea6-2fcda548eefe.gif)

**Comment by Ferkonet** [comment]
Did you find a solution?
